src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/Vedtaksform.tsx

```diff
@@ -201,6 +209,25 @@ export const Vedtaksform: React.FC<{
         // eslint-disable-next-line
     }, [behandlingErRedigerbar]);
 
+    useEffect(() => {
+        if (beregningsresultat.status === RessursStatus.SUKSESS) {
+            const kontantstøttebeløpOverstigerUtgiftsbeløpForAllePeriode =
+                beregningsresultat.data.every(
+                    (periode) =>
+                        periode.beregningsgrunnlag.kontantstøttebeløp >
+                        periode.beregningsgrunnlag.utgifter
+                );
+            settNullUtbetalingPgaKontantStøtte(
+                kontantstøttebeløpOverstigerUtgiftsbeløpForAllePeriode
+            );
+            if (kontantstøttebeløpOverstigerUtgiftsbeløpForAllePeriode) {
+                settResultatType(EBehandlingResultat.INNVILGE_UTEN_UTBETALING);
+            } else {
+                settResultatType(EBehandlingResultat.INNVILGE);
+            }
+        }
```

Comment:
Mulig dumt spørsmål, men hvorfor må vi beregne for å vite om vi skal sette resultatet til `INNVILGE_UTEN_UTBETALING`? Er ikke `kontantsstøttebeløp` og `utgifter` tall vi har i frontend før vi gjør beregningen?

Reply:
Forskjellig perioder - så ikke helt rett frem å koble utgiftsperiode med kontantstøtteperiode. 

Mulig vi heller kan løse det med å kalkulere litt "under panseret" som en forbedring

Reply:
Vi ønsker ikke å gi bruker tilbakemelding på om det blir nullutbetaling eller ikke før hen har trykket på beregn. Bruker kommer seg uansett ikke videre fra vedtak-og-beregning-siden uten å trykke på beregn. For å sette resultatet tilbake igjen til vanlig ÌNNVILGE` må brukeren taste inn et kontantstøttebeløp lavere enn utgiftsbeløpet og deretter trykke på beregn igjen.

Reply:
Det gir mening. Tenkte ikke på at de er to forskjellige lister av perioder
.